templates/admin-footer.html

```diff
@@ -37,42 +37,44 @@
             $('#info').hide();
         }
 
-        function showError(a) {
-            $('#' + a).html('<i class="fa fa-exclamation"></i> Error').addClass('disabled').attr('disabled', true);
+        function showError(conversion_test) {
```

Comment:
Change `conversion_test` to `btn` to make it more descriptive (I know `a` was not really descriptive enough for this)